src/docs/taiko.mdx

```diff
@@ -28,29 +28,27 @@ links:
 
 <Section title="Overview">
 
-    > ⚠️ The rollup is still in development. Information presented here may change by the time Mainnet is launched.
-
     Taiko aims to be a based rollup and Type 1 (Ethereum-equivalent) ZK-Rollup. The current design uses decentralised and permissionless sequencers and provers. The zkEVM circuits are based on the Ethereum Foundation's Privacy and Scaling Group research work on zkEVMs, where the team is active contributor.
    
     ###### Focus
 
     - Implementing circuits for verifying all OPCODEs and Precompiles. The execution of some components of the EVM is currently not verified by the circuits.
-    - Finalising development and launching Taiko Mainnet.
-
 
-    <Labels labels={['ZK-Rollup', 'EVM', 'Testnet']} title="Taiko" />
+    <Labels labels={['ZK-Rollup', 'EVM', 'Mainnet']} title="Taiko" />
 
 </Section>
 
 <Section title="General">
 
     <Parameter name="Block Time" value="On demand*" tooltip="The rate at which the rollup produces blocks. Blocks are produced on demand."/>
 
-    <Parameter name="Gas Limit" value="8.18 million" tooltip="The gas limit that can be consumed by an L2 block. 180 000 gas is reserved for the anchor transaction."/>
+    <Parameter name="Gas Limit" value="240,250,000" tooltip="The gas limit that can be consumed by an L2 block. 180 000 gas is reserved for the anchor transaction."/>
+
+    <Parameter name="Sequencing Frequency" value="12 - 60 seconds" tooltip="The frequency at which the rollup posts L2 transactions on Ethereum L1" />
 
-    <Parameter name="Objective Finality" value="~90 minutes" tooltip="The time it takes for a validity proof of an L2 block to be generated and submitted after it is sequenced on L1." />
+    <Parameter name="Objective Finality" value="~1.5 minutes" tooltip="The time it takes for a validity proof of an L2 block to be generated and submitted after it is sequenced on L1." />
```

Comment:
Actually within 1.5 minutes the proof that is provided is not a ZK proof. Can you verify the current proving mechanism?